app/src/main/java/se/nullable/flickboard/ui/Settings.kt

```diff
@@ -886,6 +887,7 @@ enum class LetterLayerOption(override val label: String, val layout: Layout) : L
     German("German (MessagEase)", DE_MESSAGEASE),
     GermanEnglish("German/English (MessagEase)", EN_DE_MESSAGEASE),
     Russian("Russian (MessagEase)", RU_MESSAGEASE),
+    Russian("Russian phonetic (MessagEase)", RU_PHONETIC_MESSAGEASE),
```

Comment:
```suggestion
    RussianPhonetic("Russian phonetic (MessagEase)", RU_PHONETIC_MESSAGEASE),
```

This key needs to be unique and stable, it's what's actually saved and loaded.